addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6673,10 +6673,6 @@ registry.ImageTools = ImageHandlerOption.extend({
     notify(name) {
```

Comment:
Commit message:
- > This commit adds the ability to

  ability -> possibility

- In the first paragraph, add that it is not only for the effects themselves but also for their options (like intensity, color, width) :wink: